src/js/components/sharedComponents/autocomplete/Autocomplete.jsx

```diff
@@ -79,7 +95,9 @@ export default class Autocomplete extends React.Component {
 
         target.addEventListener('blur', () => {
```

Comment:
This `fn` setUpAutocomplete should be decomposed into two functions `handleBlur` and `handleOnKeyDown`. 

We also should avoid using `addEventListener` in favor of JSX event attributes wherever possible.

I defer to your judgment as to whether to address this tech debt in your PR!

Reply:
Planning to address this as part of the Autocomplete refactor for [DEV-2921](https://federal-spending-transparency.atlassian.net/browse/DEV-2921)